### DISCUSSION TITLE
Support of RISC-V Packed SIMD Instructions

### DISCUSSION THREAD

**Discussion by tirumalnaidu** [gem5/gem5#1606] [gem5-dev]
I’ve been following the gem5 Bootcamp 2022, and in one of the tutorials, there's an example of [adding a packed SIMD instruction](https://youtu.be/Z5B02jkNpck?list=PL_hVbFs_loVSaSDPr1RJXP5RRFWjBMqq3&t=6562) to the RISC-V ISA. I wanted to check if support for this instruction extension has already been added into Gem5. I wasn't able to find any relevant information in the GitHub issues or pull requests.

**Comment by NischalJoshi777** [comment]
Hi, Tirumal  Any update on this ?
